### PR TITLE
Tweak info time format

### PR DESCRIPTION
**Pull request by TomSweeneyRedHat**
Signed-off-by: TomSweeneyRedHat <tsweeney@redhat.com>

This change touches up the way `podman info` represented the uptime.  Before it was:

```
  os: linux
  uptime: 464h26m52.62s
```
With this change it is:
```
  os: linux
  uptime: 464h 26m 52.62s (Approximately 19.33 days)

(on a newly created machine)
  os: linux
  uptime: 20m 2.53s
```

As I believe @mheon mentioned in a previous review, coming up with an exact number for the days is tricky at best due to a day being 23, 24, or 25 hours long.  Plus the odd leap second, minute, or what have you.  So I chickened out and went with the approximation.  It's just an in the neighborhood thing and I find it handy myself.

This PR originally had a change to the CLI help to remove a duplicate 'build', @sjenning beat me to the punch on that one in #413

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
This should wrap up #353

File: libpod/info.go [hunk 1]
```diff
@@ -51,7 +52,29 @@ func (r *Runtime) hostInfo() (map[string]interface{}, error) {
...
+			}
+		}
+		if !hoursFound {
+			hourBuffer.WriteRune(elem)
+		}
+	}
+
+	info["uptime"] = timeBuffer.String()
+	if hoursFound {
+		hours, _ := strconv.ParseFloat(hourBuffer.String(), 64)
```
Comment (mheon) [inline libpod/info.go]: Can you do a `strings.Split("h")` here, and take the first element of the array to get only hours? That would guarantee you don't parse the rest

Comment (TomSweeneyRedHat) [reply libpod/info.go]: @mheon I originally thought that doing the splitting might create a bunch of unnecessary strings and besides playing with a rune seemed like more fun!  I've just coded up the same functionality below using strings.Split().  I could probably tighten that up a bit more and make the naming better, but that's the gist of it.  

Let me know which approach you'd prefer and I'll finalize it and get a new PR pushed.

Comment (mheon) [reply libpod/info.go]: My main goal is to get rid of the ignored error from ParseFloat() - to make sure we don't accidentally parse garbage and output the result. I suspected that ParseFloat() would error if it encountered non-float things, like a "h".
Given choice between the two, I think the first implementation is better, though i would like the unhandled error in ParseFloat handled. Sorry for the runaround on this one

Comment (TomSweeneyRedHat) [reply libpod/info.go]: @mheon np, fun little experiment.  I'll work on handling the ParseFloat() error then.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**